### PR TITLE
Newsletter-294: translate into Chinese

### PR DESCRIPTION
**Pull request by hulatown**
TODO: need to fix some links before pushing upstream when 293 is merged

### CODE REVIEW

File: _posts/zh/newsletters/2024-03-20-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  Falbesoner 最初的工作是将用 Python 编写的 BIP324 函数整合到 Bitcoin Core 的测试套件中，结果导致代理 “非常慢，而且容易受到侧信道攻击[并且]现在不建议将其用于测试以外的任何用途”。不过，他正在用 Rust 重写该代理，并可能将其部分或全部功能作为库提供给轻客户端或其他希望本地支持 v2 比特币 P2P 协议的软件。
+
+- **BTC Lisp 概述：**Anthony Towns 在 Delving Bitcoin 上[发布][towns lisp]了他在过去几年中为比特币创建 [Lisp][] 语言变体的实验，名为 BTC Lisp。之前的讨论见周报 [#293][news293 lisp] 和 [#191][news191 lisp]。这篇文章非常详细，我们鼓励对这个想法感兴趣的人直接阅读。我们将简要引述其_结论_和_未来工作_部分：
+
+  “[BTC Lisp] 在链上可能有点昂贵，但似乎你可以做几乎任何事情[……]我不认为实现 Lisp 解释器或需要与之配套的一系列操作码太难，[但]如果没有编译器将较高层次的表示转换为共识级的操作码，编写 Lisp 代码是非常恼人的，[尽管]看起来可行。[这]项工作的下一步，[可通过]实现类似这样的语言来推进，并将其部署到 Signet/inquisition 上”。
+
+  [Simplicity][topic simplicity] 语言未来也可能被视为共识脚本语言的替代语言。Russell O'Connor 是该语言的开发者，他在[回答][oconnor lisp]中对比特币当前的脚本语言、Simplicity 和 Chia/BTC Lisp 进行了一些比较。他总结道：“Simplicity 和 clvm（[Chia Lisp 虚拟机]）都是低级语言，旨在方便机器求值，这就造成了人类难以阅读的取舍。它们是被有意设计为从不同的、人类可读的、非共识关键的语言编译而成。Simplicity 和 clvm 用不同的方式表达了同样的老事情：从环境中获取数据、对数据位进行元组化处理、运行条件语句以及各种原始操作。[……] 既然我们希望这种[高效的低级共识语言和高级的非共识可理解语言之间的分裂]无关紧要，那么低级语言的细节就变得不那么重要了。也就是说，只要稍加努力，你的高级 BTC lisp 语言也许可以翻译/编译成 Simplicity 语言[……]同样的，无论[基于 Simplicity 的] [高级非共识语言] Simphony 的设计最终走向何方，它也许都可以翻译/编译[成]你的低级 BTC lisp 语言，每种翻译/编译语言对都要约提供不同的潜在复杂性/优化机会”。
```
Comment (editor-Ajian) [inline _posts/zh/newsletters/2024-03-20-newsletter.md]:
```suggestion
  [Simplicity][topic simplicity] 语言未来也可能被视为共识脚本语言的替代语言。Russell O'Connor 是该语言的开发者，他在[回答][oconnor lisp]中对比特币当前的脚本语言、Simplicity 和 Chia/BTC Lisp 进行了一些比较。他总结道：“Simplicity 和 clvm（[Chia Lisp 虚拟机]）都是低级语言，旨在方便机器求值，这就造成了人类难以阅读的取舍。它们是被有意设计为从不同的、人类可读的、非共识关键的语言编译而成。Simplicity 和 clvm 用不同的方式表达了同样的老事情：从环境中获取数据、对数据位进行元组化处理、运行条件语句以及各种原始操作。[……] 既然我们希望这种[高效的低级共识语言和高级的非共识可理解语言之间的分裂]无关紧要，那么低级语言的细节就变得不那么重要了。也就是说，只要稍加努力，你的高级 BTC lisp 语言也许可以翻译/编译成 Simplicity 语言[……]同样的，无论[基于 Simplicity 的] [高级非共识语言] Simphony 的设计最终走向何方，它也许都可以翻译/编译[成]你的低级 BTC lisp 语言，每种翻译/编译语言对都提供了不同的潜在复杂性/优化机会”。
```

File: _posts/zh/newsletters/2024-03-20-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+- **用于轻客户端的 BIP324 代理：** Sebastian Falbesoner 在 Delving Bitcoin 上[发布][falbesoner bip324]了一个 TCP 代理，用于在版本 1（v1）比特币 P2P 协议和 [BIP324][] 中定义的 [v2 协议][topic v2 p2p transport]之间进行转换。这特别是为了让 v1 编写的轻客户端钱包得以利用上 v2 的流量加密。
+
+  轻客户端通常只公告属于自己钱包的交易，因此任何能够窃听未加密 v1 连接的人都能合理地断定轻客户端发送的交易属于使用原 IP 地址的人。在使用 v2 加密时，只有接收交易的完整节点才能确定交易来自轻客户端的 IP 地址，前提是轻客户端连接没有受到中间人攻击（在某些情况下有可能检测到中间人攻击，[以后的升级][topic countersign]可能会自动防御中间人攻击）。
+
+  Falbesoner 最初的工作是将用 Python 编写的 BIP324 函数整合到 Bitcoin Core 的测试套件中，结果导致代理 “非常慢，而且容易受到侧信道攻击[并且]现在不建议将其用于测试以外的任何用途”。不过，他正在用 Rust 重写该代理，并可能将其部分或全部功能作为库提供给轻客户端或其他希望本地支持 v2 比特币 P2P 协议的软件。
+
+- **BTC Lisp 概述：**Anthony Towns 在 Delving Bitcoin 上[发布][towns lisp]了他在过去几年中为比特币创建 [Lisp][] 语言变体的实验，名为 BTC Lisp。之前的讨论见周报 [#293][news293 lisp] 和 [#191][news191 lisp]。这篇文章非常详细，我们鼓励对这个想法感兴趣的人直接阅读。我们将简要引述其_结论_和_未来工作_部分：
+
+  “[BTC Lisp] 在链上可能有点昂贵，但似乎你可以做几乎任何事情[……]我不认为实现 Lisp 解释器或需要与之配套的一系列操作码太难，[但]如果没有编译器将较高层次的表示转换为共识级的操作码，编写 Lisp 代码是非常恼人的，[尽管]看起来可行。[这]项工作的下一步，[可通过]实现类似这样的语言来推进，并将其部署到 Signet/inquisition 上”。
```
Comment (freeyao) [inline _posts/zh/newsletters/2024-03-20-newsletter.md]:
```suggestion
  “[BTC Lisp] 在链上可能有点昂贵，但似乎你可以做几乎任何事情[……]我不认为实现 Lisp 解释器或需要与之配套的一系列操作码太难，[但]如果没有编译器将高级表示转换为共识级的操作码，编写 Lisp 代码是非常恼人的，[尽管]看起来可行。[这]项工作的下一步，[可通过]实现类似这样的语言来推进，并将其部署到 Signet/inquisition 上”。
```

Comment (freeyao) [reply _posts/zh/newsletters/2024-03-20-newsletter.md]: High-level language 一般译作高级语言

File: _posts/zh/newsletters/2024-03-20-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  在[最近的一篇博客][bitgo blog]中，BitGo 宣布在其钱包和 API 中支持使用[手续费替换（RBF）][topic rbf]来提高手续费。
+
+- **Phoenix 钱包 v2.2.0 发布：**
+  通过此版本，Phoenix 现在可以支持[通道拼接][topic splicing]，同时使用静止协商协议进行闪电网络支付（参见[周报 #262][news262 eclair2680]）。此外，Phoenix 还通过使用其 [swaproot][swaproot blog] 协议改进了换入功能的隐私和手续费。
+
+- **Bitkey 硬件签名设备发布：**
+  [Bitkey][bitkey website] 设备旨在与移动设备和 Bitkey 服务器密钥一起用于 2-3 多重签名设置。固件和各种组件的源代码目前[已可用][bitkey github]，是基于修改后的 MIT 许可协议（Commons Clause modified MIT License）。
+
+- **Envoy v1.6.0 发布：**
+  该[版本][envoy blog]增加了手续费提升交易和取消交易的功能，都基于手续费替换（RBF）。
```
Comment (freeyao) [inline _posts/zh/newsletters/2024-03-20-newsletter.md]:
```suggestion
  该[版本][envoy blog]增加了提升交易手续费和取消交易的功能，都基于手续费替换（RBF）。
```

File: _posts/zh/newsletters/2024-03-20-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+_本周的重大变更有：[Bitcoin Core][bitcoin core repo]、[Core
+Lightning][core lightning repo]、[Eclair][eclair repo]、[LDK][ldk repo]、
+[LND][lnd repo]、[libsecp256k1][libsecp256k1 repo]、[Hardware Wallet
+Interface (HWI)][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、[BTCPay
+Server][btcpay server repo]、[BDK][bdk repo]、[Bitcoin Improvement
+Proposals (BIPs)][bips repo]、[Lightning BOLTs][bolts repo]、
+[Bitcoin Inquisition][bitcoin inquisition repo] 和 [BINANAs][binana
+repo]。_
+
+*注意：下面提到的对 Bitcoin Core 的提交适用于其主开发分支，因此这些改动可能要到即将发布的 27 版本发布大约六个月后才会发布。*
```
Comment (freeyao) [inline _posts/zh/newsletters/2024-03-20-newsletter.md]:
```suggestion
*注意：下面提到的对 Bitcoin Core 的提交适用于其主开发分支，因此这些改动可能要到即将发布的 27 版本发布后大约六个月才会发布。*
```

**Review by freeyao — APPROVED** [review]
- "高级表达" to "higher level representation"
- word order adjustments

**Pull request merged**